beets/ui/commands.py

```diff
@@ -801,6 +830,53 @@ def should_resume(self, path):
                            "was interrupted. Resume (Y/n)?"
                            .format(displayable_path(path)))
 
+    def _get_plugin_options(self, task):
```

Comment:
Cool! *Choice* seems like a good name, since it avoids overlap with "configuration option" or "command-line option," as in a flag. I can't think of anything better at the moment.